src/util/ac-mld-uc.jl

```diff
@@ -1,21 +1,36 @@
-
-"implements a scalable huristic solution to the AC-MLD problem"
-function run_ac_mld_uc(case::Dict{String,<:Any}, solver; modifications::Dict{String,<:Any}=Dict{String,Any}("per_unit" => case["per_unit"]), setting::Dict{String,<:Any}=Dict{String,Any}(), int_tol::Real=1e-6)
+"implements a scalable heuristic solution to the AC-MLD problem"
+function run_ac_mld_uc(
+    case::Dict{String,<:Any},
+    solver;
+    modifications::Dict{String,<:Any}=Dict{String,Any}("per_unit" => case["per_unit"]),
+    setting::Dict{String,<:Any}=Dict{String,Any}(),
+    int_tol::Real=1e-6,
+    largest_component_only::Bool=true,
+)
     base_case = case
     case = deepcopy(case)
     _PM.update_data!(case, modifications)
 
+    # Note that this can hide some load shed if a connected component with
+    # load but no generation is created.
     _PM.simplify_network!(case)
```

Comment:
Is this true? We add deactivated buses back to the solution, but don't seem to re-add loads attached to them.